ui/app/components/job-editor.js

```diff
@@ -110,9 +110,13 @@ export default class JobEditor extends Component {
   }
 
   @action
-  updateCode(value) {
+  updateCode(value, type = 'job') {
```

Comment:
This would be a great spot to include some jsdoc comments, if type can only ever equal `job` or `hclVars`!

```
  /**
   * @param {string} value
   * @param {("job" | "hclVars")} job
   */
```

or something like that